developer_setup.md

```diff
@@ -19,9 +19,8 @@
   sudo dnf -y install postgresql-devel postgresql-server # PostgreSQL Database server and to build 'pg' Gem
   sudo dnf -y install bzip2 libffi-devel readline-devel  # For rbenv install 2.2.0 (might not be needed with other Ruby setups)
   sudo dnf -y install libxml2-devel libxslt-devel patch  # For Nokogiri Gem
-  sudo dnf -y install gcc-c++                            # For event-machine Gem
   sudo dnf -y install sqlite-devel                       # For sqlite3 Gem
-  sudo dnf -y install nodejs npm                         # For ExecJS Gem and bower
+  sudo dnf -y install nodejs                             # For ExecJS Gem and bower
```

Comment:
meh, apparently the dnf behaves differently to yum in some [cases](https://bugzilla.redhat.com/show_bug.cgi?id=1096506). Sweet! :candy:

That is because epoch and architecture are not handled consistently. (nodejs is arch, npm is noarch, then there are virtual provides and virtual requires in arch and noarch packages, etc). See [fedora mailing-list](https://lists.fedoraproject.org/archives/list/nodejs@lists.fedoraproject.org/thread/ZVMIY5LM5HXGWCP7FTZCLTAAEKPVHNMW/) for more info.

At this point (when the fedoras are broken), I can think only of this hairy :neckbeard: solution  :hammer:
```
dnf install nodejs
rpm -q --whatprovides npm || dnf install npm
```